phpBB/adm/style/permission_mask.html

```diff
@@ -14,7 +14,7 @@
 	<!-- ENDIF -->
 // ]]>
 </script>
-<script type="text/javascript" src="style/permissions.js"></script>
+<!-- INCLUDEJS permissions.js -->
```

Comment:
permissions works and I didn't find any occurrence of direct javascript inclusion of this kind (that was the issue). So I think it's fine in the acp.